src/eventHandlers/JobEventHandler.js

```diff
@@ -65,6 +67,55 @@ async function processUpdate (payload) {
   await cancelJob(payload)
 }
 
+/**
+ * Process job create event.
+ *
+ * @param {Object} payload the event payload
+ * @returns {undefined}
+ */
+async function processCreate (payload) {
```

Comment:
Can we please, create a separate function to send notifications and call it inside `processCreate` like we do in other places. This way we separate functionality for each kind of logic.